<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="loadFile_Btn">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>270</y>
     <width>325</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Загрузить исходные данные из файла</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ProcessData_Btn">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>310</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Обработать данные</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_Btn">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>350</y>
     <width>324</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить обработанные данные в файл</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_Btn">
   <property name="geometry">
    <rect>
     <x>173</x>
     <y>310</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>390</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>321</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Подсчитать количество отрицательных&lt;/p&gt;&lt;p align=&quot;center&quot;&gt; элементов в таблице и увеличить на это&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;значение минимальный и максимальный элементы таблицы .&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
